seacrowd/sea_datasets/oscar_2201/oscar_2201.py

```diff
@@ -0,0 +1,236 @@
+}
+
+@inproceedings{OrtizSuarezSagotRomary2019,
+    author    = {Pedro Javier {Ortiz Su{'a}rez} and Benoit Sagot and Laurent Romary},
+    title     = {Asynchronous pipelines for processing huge corpora on medium to low resource infrastructures},
+    series = {Proceedings of the Workshop on Challenges in the Management of Large Corpora (CMLC-7) 2019. Cardiff, 22nd July 2019},
+    editor    = {Piotr Bański and Adrien Barbaresi and Hanno Biber and Evelyn Breiteneder and Simon Clematide and Marc Kupietz and Harald L{"u}ngen and Caroline Iliadi},
+    publisher = {Leibniz-Institut f{"u}r Deutsche Sprache},
+    address   = {Mannheim},
+    doi       = {10.14618/ids-pub-9021},
+    url       = {http://nbn-resolving.de/urn:nbn:de:bsz:mh39-90215},
+    pages     = {9 -- 16},
+    year      = {2019},
+    abstract  = {Common Crawl is a considerably large, heterogeneous multilingual corpus comprised of crawled documents from the internet, surpassing 20TB of data and distributed as a set of more than 50 thousand plain text files where each contains many documents written in a wide variety of languages. Even though each document has a metadata block associated to it, this data lacks any information about the language in which each document is written, making it extremely difficult to use Common Crawl for monolingual applications. We propose a general, highly parallel, multithreaded pipeline to clean and classify Common Crawl by language; we specifically design it so that it runs efficiently on medium to low resource infrastructures where I/O speeds are the main constraint. We develop the pipeline so that it can be easily reapplied to any kind of heterogeneous corpus and so that it can be parameterised to a wide range of infrastructures. We also distribute a 6.3TB version of Common Crawl, filtered, classified by language, shuffled at line level in order to avoid copyright issues, and ready to be used for NLP applications.},
+    language  = {en}
+}
+"""
+
+_DATASETNAME = "oscar_2201"
+_DESCRIPTION = """\
+OSCAR or Open Super-large Crawled Aggregated coRpus is a huge multilingual corpus
+obtained by language classification and filtering of the Common Crawl corpus using
+the ungoliant architecture. Data is distributed by language in both original and
+deduplicated form.
+"""
+
+_HOMEPAGE = "https://huggingface.co/datasets/oscar-corpus/OSCAR-2201"
+_LICENSE = """\
```

Comment:
For the license, to follow the standard of custom license defined in `template/templates.py`
```
# Some datasets may also have custom licenses. In this case, simply put f'{Licenses.OTHERS.value} | {FULL_LICENSE_TERM}' into `_LICENSE`
```

we can add the the `Licenses.OTHERS.value} | ` before the custom licensing terms

Reply:
I was actually confused by this because there is no Licenses.OTHERS in `constants.py`. I thought `|` is `or`.
Let me add a new one in `constants.py` then.

Reply:
Hi @akhdanfadh, the adjustment is already on the review (https://github.com/SEACrowd/seacrowd-datahub/pull/124), feel free to comment/adjust/test it for your dataloader implementation.

Reply:
Oh I just saw this. I think the value for `Licenses.OTHERS` should be `Custom License (others)` (see my recent commit below) as it will be different among various datasets. Then you can add the custom licenses text after `|`.

Reply:
yea I mixed up in that part, thanks for the revising!

Reply:
yea I mixed up in that part, thanks for the revision!

Reply:
Currently waiting for #124 to avoid merge conflict.

Reply:
Revert commit and only fix the citation part. Waiting for #124 to be merged.

Reply:
@akhdanfadh @sabilmakbar , we have just merged https://github.com/SEACrowd/seacrowd-datahub/pull/124. Please update the code accordingly 👍🏻